### DISCUSSION TITLE
Support for macOS Monterey

### DISCUSSION THREAD

**Discussion by ghost** [noah-nuebling/mac-mouse-fix#167] [Questions] [answered]
Will the app be updated to support macOS Monterey soon?

**Comment by noah-nuebling** [comment]
I have a lotta stress with college at the moment and don't have time to work on the project sorry :/

How is it incompatible with Monterey?

**Reply by noah-nuebling** [reply]
I should be able to put some work in around the middle of December

**Reply by ghost** [reply]
Ah, good luck with college! The app does seem to be working on Monterey so far, but it does seem a little slower and not as sharp as it was on Big Sur. But maybe that's just me?

**Reply by noah-nuebling** [reply]
Thanks! That's interesting I'll look into it

**Reply by Anywhere-Music-Player** [reply]
For me it's perfectly working on macOS 12.1 (21C52)!

**Comment by noah-nuebling** [comment, marked as answer]
Marking this as answered assuming it's been resolved. Let me know if you have more issues!
